src/components/CurriculumComponents/CurricTimetablingNewView/index.tsx

```diff
@@ -1,18 +1,92 @@
 "use client";
-
 import {
   OakFlex,
   OakHeading,
+  OakP,
   OakPrimaryButton,
 } from "@oaknational/oak-components";
 
 export const CurricTimetablingNewView = () => {
+  const DEFAULT_LESSONS = 10;
+
   return (
-    <OakFlex $flexDirection={"column"}>
-      <OakHeading tag="h1" $font="heading-2">
-        New timetable
+    <OakFlex $flexDirection={"column"} $gap={"space-between-l"}>
+      <OakHeading tag="h2" $font="heading-2">
+        Enter lessons per term
       </OakHeading>
-      <OakPrimaryButton element="a" href="/timetabling/name">
+      <OakP $font={["body-2", "body-1"]}>
+        We recommend calculating the lessons per term by the actual teaching
+        time, taking into account any other activities beyond the curriculum
+        such as school trips.
+      </OakP>
+
+      <OakFlex $flexDirection={"column"} $gap={"space-between-l"}>
+        <OakFlex
+          $flexDirection={"column"}
+          $gap={"space-between-s"}
+          $maxWidth={["100%", "all-spacing-18"]}
+        >
+          <OakHeading id="autumn-heading" tag="h3" $font="heading-3">
+            Autumn
+          </OakHeading>
+          <label htmlFor="autumn-lessons">Number of lessons</label>
+          <input
+            id="autumn-lessons"
+            name="autumn-lessons"
+            type="number"
+            value={DEFAULT_LESSONS}
+            disabled
+            aria-describedby="autumn-heading"
+            style={{ height: 60, borderRadius: 8, width: "100%" }}
+          />
+        </OakFlex>
+
+        <OakFlex
+          $flexDirection={"column"}
+          $gap={"space-between-s"}
+          $maxWidth={["100%", "all-spacing-18"]}
+        >
+          <OakHeading id="spring-heading" tag="h3" $font="heading-3">
+            Spring
+          </OakHeading>
+          <label htmlFor="spring-lessons">Number of lessons</label>
```

Comment:
Lets use `<OakJauntyAngleLabel/>` and `<OakTextInput/> so it's in keeping with oak